.golangci.yaml

```diff
@@ -0,0 +1,44 @@
+linters:
+  enable:
+    - errorlint
+    - gci
+    - gocritic
+    - gofumpt
+    - misspell
+    - nonamedreturns
+    - testifylint
+    - errcheck
+
+linters-settings:
+  errorlint:
+    # Check whether fmt.Errorf uses the %w verb for formatting errors.
+    # See the https://github.com/polyfloyd/go-errorlint for caveats.
+    errorf: true
+    # Permit more than 1 %w verb, valid per Go 1.20 (Requires errorf:true)
+    errorf-multi: true
+    # Check for plain type assertions and type switches.
+    asserts: true
+    # Check for plain error comparisons.
+    comparison: true
```

Comment:
why defining all these while they are the default values ?

https://golangci-lint.run/usage/linters/#errorlint

Reply:
To be honest I grab a file from another projet of a friend without check as a quick win to add testifylint 👀🗿😅

Reply:
Lol. Indeed

I will keep working on my side project then

https://github.com/ccoVeille/golangci-lint-config-examples

As I face many people like "oh well let's copy paste on that works" 😅😂

Reply:
Ahah 

I will take a look on your configs 👀

Reply:
Done ✔️